wikimedia/wikipedia-ios
# Article as a Living Doc Modal: Add space before external link icon

**Fixes Phabricator ticket:** https://phabricator.wikimedia.org/T266256#6628502

### Notes
* Adds some spacing.

### Test Steps
1.It may help if you adjust the computed var shouldAttemptToShowArticleAsLivingDoc to always return true, so you see the experiment on all articles.
1. Find a modal w/ an external link. Ensure there is some spacing between the text and the link.

### Screenshots/Videos 
<img width="200" alt="Screenshot 10" src="https://user-images.githubusercontent.com/9295855/99575350-5f3ea280-298d-11eb-80ec-f7d8afa5654f.png"> <img width="200" alt="Screenshot 9" src="https://user-images.githubusercontent.com/9295855/99575356-61086600-298d-11eb-89b3-7028a2727d4d.png">
